spec/features/users_can_view_reports_spec.rb

```diff
@@ -429,6 +443,19 @@ def expect_to_see_a_table_of_reports(selector:, reports:)
       end
     end
 
+    context "when the report has an export_filename" do
+      scenario "the link to download the report is the download path instead of the show path" do
+        report = create(:report, :approved, export_filename: "exported_csv", organisation: partner_org_user.organisation)
+
+        visit reports_path
+        within "##{report.id}" do
```

Comment:
Spacing here is different to the similar example earlier in this commit (in this file)